sal-service/src/main/java/org/ow2/proactive/sal/service/service/TaskBuilder.java

```diff
@@ -57,19 +57,49 @@ public class TaskBuilder {
     private static final String SCRIPTS_SEPARATION_GROOVY = NEW_LINE + NEW_LINE + "// Separation script" + NEW_LINE +
                                                             NEW_LINE;
 
+    private static final String EMS_DEPLOY_PRESCRIPT = "emsdeploy_prescript.sh";
```

Comment:
I see that you named the variables as the script names but I think it would be nice to have the same pattern for all script files variables. 
Something like *_SCRIPT would be nice.

Reply:
Done